notification-service/src/main/java/com/programming/NotificationServiceApplication.java

```diff
@@ -0,0 +1,24 @@
+package com.programming;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@SpringBootApplication
+@Slf4j
+public class NotificationServiceApplication {
+    public NotificationServiceApplication(KafkaTemplate<String, OrderPlacedEvent> kafkaTemplate) {
+    }
+
+    public static void main(String[] args) {
+        SpringApplication.run(NotificationServiceApplication.class, args);
+    }
+
+    @KafkaListener(topics = "notificationTopic")
```

Comment:
move listener to a separate component and also write the test case separately. we dont put business logic in the main spring boot application class.